.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Build on various envs
```

Comment:
It the PR comment, you wrote:

> Adjust repository settings to changes in the workflows.

What exactly should to be done?

Reply:
In `Branch protection rules` there is a list of workflows required to pass before merging a PR. I renamed some of the workflows so that list must be updated. Also the new workflow should be added to it.